src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
@wesm  @edani  SSE/AVX does not require aligned access... nor is it likely to be a performance concern. Can you guys email me? (lemire@gmail.com) Though I am interested, I have not been following this project. I'd be interested in having a **quick** email exchange with people who are knowledgeable in ``parquet-cpp``. Then we can probably put something together...